sklearn/metrics/_pairwise_distances_reduction/_gemm_term_computer.pyx.tp

```diff
@@ -31,7 +31,7 @@ from ...utils._cython_blas cimport (
 
 {{for name_suffix, upcast_to_float64, INPUT_DTYPE_t, INPUT_DTYPE in implementation_specific_values}}
 
-cdef class GEMMTermComputer{{name_suffix}}:
+cdef class MiddleTermComputer{{name_suffix}}:
     """Component for `FastEuclidean*` variant wrapping the logic for the call to GEMM.
 
     `FastEuclidean*` classes internally compute the squared Euclidean distances between
```

Comment:
Reminder: we will need to adapt docstring here and potentially in other places, but let's keep that for later.